public/app/percona/dbaas/components/MultipleActions/MultipleActions.tsx

```diff
@@ -10,14 +10,24 @@ export const MultipleActions: FC<MultipleActionsProps> = ({ actions, disabled, d
   const styles = getStyles(theme);
 
   const Toggle = React.forwardRef<HTMLButtonElement>((props, ref) => (
-    <IconButton name="ellipsis-v" size="xl" disabled={disabled} data-testid={dataTestId} ref={ref} {...props} />
+    <span className={styles.iconWrapper}>
+      <IconButton name="ellipsis-v" size="xl" disabled={disabled} data-testid={dataTestId} ref={ref} {...props} />
+    </span>
   ));
 
   return (
     <Dropdown toggle={Toggle} data-testid="multiple-actions-dropdown">
-      {actions.map(({ title, action, disabled }) => (
-        <Fragment key={title}>
-          {disabled ? <span className={styles.disabledButton}>{title}</span> : <span onClick={action}>{title}</span>}
+      {actions.map(({ title, action, disabled }, index) => (
+        <Fragment key={index}>
```

Comment:
Let's not use index as key.
[Source](https://reactjs.org/docs/lists-and-keys.html#:~:text=We%20don't%20recommend%20using,an%20index%20as%20a%20key.)

Reply:
I know that it is not recommended to use it, but in this case we are not changing list in run time so it shouldn't be problem. What else do you recommend to use? Since title is reactNode now and we don't have any react key generator package in pmm.

Reply:
Oh, title is no longer text, forgot that.
In that case, let's stick to index, no need to generate keys.
Can we change this prop to something more appropriate than `title`? Maybe `content`?

Reply:
Renamed title to content.